packages/ui5-info/test/ui5-theme-info.test.ts

```diff
@@ -36,8 +38,41 @@ describe('getUi5Themes', () => {
         expect(getUi5Themes('1.72')).toEqual(allExpectedThemes.slice(0, 3));
         expect(getUi5Themes('1.101')).toEqual(allExpectedThemes.slice(0, 3));
         expect(getUi5Themes('1.102')).toEqual(allExpectedThemes);
+        expect(getUi5Themes('1.135.0')).toEqual(allExpectedThemes);
+        // Filter out sap_belize from allExpectedThemes
+        const allExpectedThemesExcludingBelize = allExpectedThemes.filter((theme) => theme.id !== 'sap_belize');
```

Comment:
This modification of the suite test data is introducing potential problem with test isolation. Should restore the test data to original state between every test now, otherwise running individual tests can result in failures.